docs/schema.md

```diff
@@ -527,6 +538,7 @@ The defined ecosystems are:
 | `ConanCenter` | The ConanCenter ecosystem for C and C++; the `name` field is a Conan package name.  |
 | `Rocky Linux` | The Rocky Linux package ecosystem; the `name` is the name of the source package. The ecosystem string might optionally have a `:<RELEASE>` suffix to scope the package to a particular Rocky Linux release. `<RELEASE>` is a numeric version.
 | `AlmaLinux` | AlmaLinux package ecosystem; the `name` is the name of the source package. The ecosystem string might optionally have a `:<RELEASE>` suffix to scope the package to a particular AlmaLinux release. `<RELEASE>` is a numeric version.
+| `Bitnami` | Bitnami package ecosystem; the `name` is the name of the affected component. |
```

Comment:
Is there documentation or additional rules around the naming for a "component" ?

Reply:
We don't have any special rule for naming our components, apart from using lowercase and dashes as separators (i.e. `wordpress-multisite`). We also publish the official URL of those packages in our [Dockerfiles](https://github.com/bitnami/containers/blob/main/bitnami/wordpress/6/debian-11/Dockerfile#L23) on GitHub. 

We also provide `purl` for those packages like `pkg:bitnami/wordpress@6.2.2`.

Reply:
Ack, thanks!